libraries/exoplayer/src/main/java/androidx/media3/exoplayer/LoadControl.java

```diff
@@ -80,6 +81,18 @@ final class Parameters {
      */
     public final long targetLiveOffsetUs;
 
+    /**
+     * Sets the time at which the last rebuffering occurred, in milliseconds since boot including
+     * time spent in sleep.
+     *
+     * <p>The time base used is the same as that measured by {@link SystemClock#elapsedRealtime}.
+     *
+     * <p><b>Note:</b> If rebuffer events are not known when the load is started or continued, or if
+     * no rebuffering has occurred, or if there have been any user interactions such as seeking or
+     * stopping the player, the value will be set to {@link C#TIME_UNSET}.
+     */
+    public final long lastRebufferRealtimeMs;
```

Comment:
Please make this `Us` so it is consistent to the other position units we use in LoadControl.Parameters

Reply:
I guess the difference between this and `playbackPositionUs` & `bufferedDurationUs` is that this value is measured in the `SystemClock.elapsedRealtimeMs` timebase, rather than the 'media timebase' (for which we pretty consistently use microseconds inside ExoPlayer). 

We seem to have a mixture of micros and millis throughout out the codebase for this:
* https://github.com/search?q=repo%3Aandroidx%2Fmedia%20realtimeus&type=code (36 files)
* https://github.com/search?q=repo%3Aandroidx%2Fmedia%20realtimems&type=code (41 files)

Reply:
These values are already kind of (deliberately) 'inconsistent' with each other by being measured in different time bases anyway, whether they use the same units or not. As in, it would probably not be meaningful to add/subtract/compare `lastRebufferRealtimeUs` with `playbackPositionUs`.

I'm not saying we should definitely use millis or micros, just that there's arguments for both I think.

Reply:
Yeah, I think I see you point of the last buffer timestamp being taken from the `clock.elapsedRealtime()` which is MS. Looking at the arguments of `EPII.updateRebufferingState(boolean isRebuffering, boolean resetLastRebufferRealtimeMs)` and the naming of the boolean flag I'm not sure how deliberate this was. 

But looks like falling into a democratic mode is quickest to resolve this dispute which makes me loose 2:1. :)